tests/env/support/invalid_keys.yml

```diff
@@ -1,6 +1,6 @@
 names: nlp
 chanels:
   - anaconda
-dependecies:
+dependencies:
```

Comment:
Same here, this is a change in behaviour caused by enforcing `dependencies` being a required field.